lib/recipient.rb

```diff
@@ -0,0 +1,40 @@
+require 'httparty'
+
+class Recipient
+  
+  attr_reader :slack_id, :name
+  
+  def initialize(slack_id:, name:)
+    @slack_id = slack_id
+    @name = name
+  end
+  
+  def send_message(message)
+    #send message using HTTParty
+    response = HTTParty.post("https://slack.com/api/chat.postMessage", body: {token: ENV['SLACK_TOKEN'], text: message, channel: @slack_id})
+    if response.code != 200 || response['ok'] == false
+      raise SlackAPIError
+    end
+    
+    return true
+    
+  end
+  
+  def self.get(url)
+    response = HTTParty.get(url, query: {token: ENV['SLACK_TOKEN']})
+    if response.code != 200 || response['ok'] == false
+      raise SlackAPIError
+    end
```

Comment:
This honestly is generally pretty sufficient! The one thing I would add is figuring out what part of the response gives a useful message explaining what went wrong and making sure to record that as a property of SlackAPIError.
If it's unclear what I mean by that, please ask!